src/Document/CategoryManager.php

```diff
@@ -206,11 +206,6 @@ protected function loadCategories(ContextInterface $context): void
             $categoryId = $category->getId();
             \assert(null !== $categoryId);
             $this->categories[$contextId][$categoryId] = $category;
-
-            $parent = $category->getParent();
-            if (null !== $parent) {
-                $parent->addChild($category);
```

Comment:
This should have been removed when we deprecated/removed the disableLazyChildren collection here: https://github.com/sonata-project/SonataClassificationBundle/pull/765